queenbee/io/common.py

```diff
@@ -410,3 +410,12 @@ class IOAliasHandler(BaseModel):
         description='Name of the function. The input value will be passed to this '
         'function as the first argument.'
     )
+
+    index: int = Field(
+        0,
```

Comment:
Any chance we can replace this a nullable field rather than a default one? This will make the recipe `digest` more reliable when comparing with digests already present on the database.

```python
index: Optional[int] = Field(None, description='An integer...'
```

Reply:
Hi @AntoineDao, Thanks for the note. I should have waited for your before merging this in. I remember we had issues with Optional when we used it in Honeybee Schema: https://github.com/ladybug-tools/honeybee-schema/issues/121#issuecomment-625302172 - optional doesn't really translate to OpenAPI and that can cause inconsistencies.

I know that this might generate a new hash for the current packages the first time but it will be fine after that.

Reply:
I think waiting would have been helpful here. I'm a bit nervous about what will happen when we merge this into pollination-server so I will push it now and we can keep an eye on it to see if it breaks anything

Reply:
Sounds good! Based on what we talked about before - adding new optional fields should be fine but you never know. 🤞